src/app.js

```diff
@@ -146,30 +170,44 @@ export default async function ({
       }
 
       if (queueItem && queueItem.importJobState[operation] === IMPORT_JOB_STATE.IN_QUEUE) {
-        logger.debug(`Found item in queue to be imported ${queueItem.correlationId}`);
+        logger.debug(`Found item in q ueue to be imported${queueItem.correlationId} ${waitTimePrint(waitSinceLastOp)}`);
         await mongoOperator.setState({correlationId: queueItem.correlationId, state: QUEUE_ITEM_STATE.IMPORTER.IMPORTING});
         return true;
       }
 
       if (queueItem && queueItem.importJobState[operation] === IMPORT_JOB_STATE.EMPTY) {
-        logger.debug(`Found item in queue to be imported ${queueItem.correlationId} but importJobState for ${operation} is ${queueItem.importJobState[operation]}`);
-        logger.debug(JSON.stringify(queueItem.importJobState));
+        logger.debug(`Found item in queue to be imported ${queueItem.correlationId} ${waitTimePrint(waitSinceLastOp)} but importJobState for ${operation} is ${queueItem.importJobState[operation]}`);
+        logger.silly(JSON.stringify(queueItem.importJobState));
 
         // check whether also the other queue is EMPTY or a final state
         const otherOperationImportJobState = operation === OPERATIONS.CREATE ? OPERATIONS.UPDATE : OPERATIONS.CREATE;
         const otherOperationImportJobStateResult = queueItem.importJobState[otherOperationImportJobState];
         logger.debug(`Checking importerJobState for other operation: ${otherOperationImportJobState}: ${otherOperationImportJobStateResult}`);
 
         if ([IMPORT_JOB_STATE.EMPTY, IMPORT_JOB_STATE.DONE, IMPORT_JOB_STATE.ERROR, IMPORT_JOB_STATE.ABORT].includes(otherOperationImportJobStateResult)) {
-          logger.debug(`Other importJob in not ongoing/pending, importing done`);
+          logger.debug(`Other importJob (${otherOperationImportJobState}) in not ongoing/pending, importing done`);
           await mongoOperator.setState({correlationId: queueItem.correlationId, state: QUEUE_ITEM_STATE.DONE});
           return true;
         }
 
+        logger.debug(`Found item in queue to be imported ${queueItem.correlationId} ${waitTimePrint(waitSinceLastOp)}`);
         await mongoOperator.setState({correlationId: queueItem.correlationId, state: QUEUE_ITEM_STATE.IMPORTER.IMPORTING});
         return true;
       }
       return false;
     }
   }
+
+  // We should move this to some commons
```

Comment:
Lets move it is melinda-backend-commons
and change it to print once in 15mins (900000ms)

Reply:
Moved to melinda-backend-commons: https://github.com/NatLibFi/melinda-backend-commons-js/pull/21 #